components/InterestZoneModal/InterestZoneView.tsx

```diff
@@ -35,7 +35,8 @@ const InterestZoneView = ({ zone, partialZone }: Props) => {
   const { setInterestZone } = useContext(InterestZoneProviderContext);
   const { AddZone, UpdateZone, DeleteZone } = useZoneActions(zone?._id!);
   const { t } = useTranslation('common');
-
+  const nameRegex = /^[a-zăîâșțlàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.'-]+$/ui
```

Comment:
Please look into simplifying the above regex
https://dev.to/tillsanders/let-s-stop-using-a-za-z-4a0m